### ISSUE TITLE
Az.PolicyInsights: Get-AzPolicyEvent failure on CloudShell

### ISSUE THREAD

**Issue by MiYanni** [Azure/azure-powershell#9232]
Running this cmdlet on CloudShell throws an error:
```powershell
PS /home/miyanni/clouddrive/psdrop2> Resolve-AzError -Last


   HistoryId: 9
Message        : Object reference not set to an instance of an object.
StackTrace     :    at Microsoft.Azure.Commands.PolicyInsights.Models.PolicyEvent..ctor(PolicyEvent policyEvent)
                    at Microsoft.Azure.Commands.PolicyInsights.Cmdlets.GetAzureRmPolicyEvent.<>c.<ExecuteCmdlet>b__56_0(PolicyEvent policyEvent)
                    at System.Linq.Enumerable.SelectListIterator`2.MoveNext()
                    at System.Management.Automation.MshCommandRuntime._WriteObjectsSkipAllowCheck(Object sendToPipeline)
                    at System.Management.Automation.MshCommandRuntime.DoWriteObjects(Object sendToPipeline)
                    at System.Management.Automation.MshCommandRuntime.WriteObject(Object sendToPipeline, Boolean enumerateCollection)
                    at System.Management.Automation.Cmdlet.WriteObject(Object sendToPipeline, Boolean enumerateCollection)
                    at Microsoft.WindowsAzure.Commands.Utilities.Common.AzurePSCmdlet.WriteObject(Object sendToPipeline, Boolean enumerateCollection)
                    at Microsoft.Azure.Commands.PolicyInsights.Cmdlets.GetAzureRmPolicyEvent.ExecuteCmdlet()
                    at Microsoft.WindowsAzure.Commands.Utilities.Common.AzurePSCmdlet.ProcessRecord()
Exception      : System.NullReferenceException
InvocationInfo : {Get-AzPolicyEvent}
Line           : Get-AzPolicyEvent -Debug
Position       : At line:1 char:1
                 + Get-AzPolicyEvent -Debug
                 + ~~~~~~~~~~~~~~~~~~~~~~~~
HistoryId      : 9
```

Unfortunately, when using the `Azure SDK Powershell Test` subscription and the `-Debug` parameter on the cmdlet, the output is too long to capture all of it.

**Comment by markcowl** [comment]
This looks like it is related to the upgrade to Newtonsoft.Json 12 in PowerShell Core 6.2

**Comment by markcowl** [comment]
Candidate PR to fix is here: #9233

**Comment by MiYanni** [comment]
PR has been merged.
